### ISSUE TITLE
Training time and you used the number of gpus

### ISSUE THREAD

**Issue by chengxj8888** [PRBonn/LiDiff#7]
I want to know your training time and the number of gpus, Thank you!

**Comment by nuneslu** [comment]
Hi! In total the diffusion model took around 3 days to train with two NVIDIA RTX A6000. The refinement took only one day and one GPU.

**Comment by nuneslu** [comment]
I will close this issue now, but fell free to reopen it or create a new one if necessary. :)

**Comment by t-martyniuk** [comment]
hello! as in your training config n_gpus = 1, I just wanted to make sure:
you were training on 2 GPUs with batch_size = 2, and thus 2x of the learning rate from the config, 0.0002? Am I right to assume that?

**Comment by nuneslu** [comment]
Hi @t-martyniuk! No, even with 2 GPUs we have used the same learning rate of `0.0001`.

**Comment by t-martyniuk** [comment]
thank you @nuneslu!
i'm wondering what could be an issue, because it's been now more than 3 days, and i have only 10 epochs finished (also 2 GPUs, batch size 2, but A100 40GB). I kept lr=0.0001 same as you.

**Comment by nuneslu** [comment]
It could be related to data storage or CPU processing. We were storing our data in a SSD which makes the data loading faster. Besides, the dataloader is somewhat slow since we have to deal with the ground truth maps which are quite large, so depending on the CPU usage, the processing time may vary.

**Comment by t-martyniuk** [comment]
thanks! makes sense
